packages/flutter_goldens/README.md

```diff
@@ -6,4 +6,6 @@ See also:
 ￼
  * https://skia.org/docs/dev/testing/skiagold/
  * https://flutter-gold.skia.org/
- * https://github.com/flutter/flutter/wiki/Writing-a-golden-file-test-for-package:flutter
+ * [Writing a golden file test for package flutter]
+
+[Writing a golden file test for package flutter]: /docs/contributing/testing/Writing-a-golden-file-test-for-package-flutter.md
```

Comment:
Are we ready to replace links to the wiki with links to the repo? I can undo this change if not.

More links that could be updated: https://github.com/search?q=repo%3Aflutter%2Fflutter%20https%3A%2F%2Fgithub.com%2Fflutter%2Fflutter%2Fwiki&type=code

Reply:
We are ready as of this morning! Running through to clean up these links is on my to do list for this week! Thanks for your help!